core/src/main/java/io/questdb/cutlass/auth/Authenticator.java

```diff
@@ -24,37 +24,21 @@
 
 package io.questdb.cutlass.auth;
 
-import io.questdb.cairo.SecurityContext;
 import io.questdb.network.Socket;
-import io.questdb.std.Mutable;
-import io.questdb.std.QuietCloseable;
 import org.jetbrains.annotations.NotNull;
 
-public interface Authenticator extends QuietCloseable, Mutable {
+public interface Authenticator extends AuthenticatorBase {
```

Comment:
nit: we could name this interface `SocketAuthenticator` and rename the base interface into `Authenticator`.

Reply:
done